tests/test_df_registration.py

```diff
@@ -22,80 +22,105 @@ def dataframe():
 
 
 def test_clean_names_registration(dataframe):
+    """Test if DataFrame has clean_method method"""
```

Comment:
That's really kind of you, @nvamsikrishna05, thanks a ton for doing this! :smile: